<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIMethodParamDetails</class>
 <widget class="QWidget" name="SIMethodParamDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxDetails">
     <property name="title">
      <string>Parameter Details:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::RowWrapPolicy::DontWrapRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editParamName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Parameter:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPlainTextEdit" name="textParamDescribe">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="plainText">
           <string notr="true"/>
          </property>
          <property name="overwriteMode">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string notr="true">Describe data here</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkDeprecated">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Deprecated:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="editDeprecated">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Type &amp;&amp; Default Value:</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Type:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="comboParamType"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Value:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QCheckBox" name="checkDefaultValue">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editDefaultValue"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editParamName</tabstop>
  <tabstop>comboParamType</tabstop>
  <tabstop>textParamDescribe</tabstop>
  <tabstop>checkDeprecated</tabstop>
  <tabstop>editDeprecated</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
